frontend/components/utils/FormatCurrency.vue

```diff
@@ -34,6 +36,19 @@ export default Vue.extend({
         },
     },
     computed: {
+        formattedNumber(): number {
+            if (!this.value.toFixed()) {
+                return 0;
+            }
+
+            if (this.value < 0.01) {
+                const decimalCutOff = Math.abs(Math.floor(Math.log10(Number(this.value)))) + 1;
+                return parseFloat(
+                    parseFloat(this.value.toString()).toFixed(decimalCutOff === Infinity ? 0 : decimalCutOff)
+                );
```

Comment:
As stated above and in general, please avoid numerous conversions back and forth, the number/bignumber should ideally be treated as a bignumber and converted to a string only once right before we need to display it

Reply:
Correct me If I am wrong, but as I am using normal Javascript functions to calculate this, I don't see any way around. Especially one like above where I determine the number of decimal points I need:
```js
 const decimalCutOff = Math.abs(Math.floor(Math.log10(Number(this.value)))) + 1;
```
I don't see any way to do this with BigNumbers. 

As for the line you highlighted: I only found this setting in the [BigNumber config ](https://mikemcl.github.io/bignumber.js/#exponential-at) where I can change this rounding. However I don't think its smart to change the config of BigNumber, which is why I went with traditional javascript methods.

Reply:
See my comment above to why you may not even need it